example/index.html

```diff
@@ -8,6 +8,9 @@
 </head>
 <body>
   <div id="app"></div>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
```

Comment:
these files are not needed

Reply:
I'll push another commit removing these lines, but given that react and react-dom moved to peerDependencies, it's not clear to me how the example is getting instances of React/React-DOM. I'm likely just missing something.